jwst/conftest.py

```diff
@@ -73,3 +74,27 @@ def jail(request, tmpdir_factory):
     os.chdir(str(newpath))
     yield newpath
     os.chdir(old_dir)
+
+@pytest.mark.trylast
+def pytest_configure(config):
+    terminal_reporter = config.pluginmanager.getplugin('terminalreporter')
+    config.pluginmanager.register(TestDescriptionPlugin(terminal_reporter), 'testdescription')
+
+class TestDescriptionPlugin:
+
+    def __init__(self, terminal_reporter):
+        self.terminal_reporter = terminal_reporter
+        self.desc = None
+
+    def pytest_runtest_protocol(self, item):
+        self.desc = inspect.getdoc(item.obj)
```

Comment:
It seems like our ASDF schema test items don't have `obj` attributes (see failing tests).  So perhaps something like:
```suggestion
        try:
            self.desc = inspect.getdoc(item.obj)
        except AttributeError:
            self.desc = None
```
might fix that.